### ISSUE TITLE
03-06-2019: Meeting w/ Camille, Carole

### ISSUE THREAD

**Issue by gkiar** [gkiar/analytical-stability#6]
topics: 
> - a brief overview of our project (a tiny bit outdated, but [summarized here](https://github.com/gkiar/analytical-stability/blob/master/docs/outline.md))
> - describing the model architecture we've adopted (brief paper [summary here](https://github.com/gkiar/reproreading/issues/44))
> - overview of the current performance ([notebook here](https://nbviewer.jupyter.org/github/gkiar/3d-cnn-ae/blob/master/code/applying_model.ipynb), in particular the first set of figures)
> - asking for recommendations on how to improve our performance

**Comment by gkiar** [comment]
Notes from Antoine (pre meeting):
- [ ] check performance against test and validation sets
- [ ] possibly add skip connections to increase spatial context
- [ ] decrease learning rate to take more advantage of the high number of samples/epochs

**Comment by cmaumet** [comment]
Questions/things to look at (from me):
 - Can we avoid training in the background (i.e. we know those should be zeros, no need to train the autoencoder in those regions). We could use use the MNI brainmask for that.

**Comment by cmaumet** [comment]
From Carole: There is a paper referring to square like artefacts in autoencoders. Could be related to max-pooling. Something in the architecture. Zara had the same king of artefacts but when using the latent representation not the reconstructed image.

The problem may come from the upsampling part.

**Comment by cmaumet** [comment]
Meeting w/ Carole Latizien, @efromont, @cmaumet.

**Comment by cmaumet** [comment]
<img width="558" alt="Screen Shot 2019-06-03 at 10 43 16" src="https://user-images.githubusercontent.com/5374264/58788565-6f718f00-85ec-11e9-90f8-29701a896e8a.png">

**Comment by gkiar** [comment]
https://distill.pub/2016/deconv-checkerboard/

**Comment by gkiar** [comment]
https://github.com/Microsoft/CNTK/issues/1741
